sysvinit/openbsd/cloudinitlocal.tmpl

```diff
@@ -0,0 +1,62 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudinitlocal
+{#
+``cloudinitlocal`` purposefully does not depend on ``dsidentify``.
+That makes it easy for image builders to create images without ``dsidentify``.
```

Comment:
you can delete this comment, and I should delete it from the FreeBSD rc scripts.
I've made it so that dsidentify can be easily disabled

on OpenBSD, that is easier still.